### PR TITLE
Improvements to easyrule cli script

### PR DESCRIPTION
**Pull request by oliwel**
* add method to "unblock" a former block rule
* derive name of block list alias from script, allowing management of multiple individial lists by using symlinks

### CODE REVIEW

File: etc/inc/easyrule.inc [hunk 1]
```diff
@@ -332,6 +348,87 @@ function easyrule_parse_block($int, $src, $ipproto = "inet") {
...
+			write_config();
+			$retval = filter_configure();
+			if (!empty($_SERVER['DOCUMENT_ROOT'])) {
+				header("Location: firewall_aliases.php");
+				exit;
+			} else {
+				return gettext("Host unblocked successfully");
+			}
+		} else {
+			return gettext("Host ist not on block list: " . $host);
```
Comment (phil-davis) [inline etc/inc/easyrule.inc]: s/ist/is/
I guess you have revealed your European background :)

Comment (oliwel) [inline etc/inc/easyrule.inc]: I should either add more caffeine or stop coding after midnight....

**Comment by rbgarga** [comment]
In order to get patches applied you must sign Individual or Corporate CLA as described at https://www.pfsense.org/about-pfsense/#legal

**Comment by oliwel** [comment]
CLA signed

**Comment by oliwel** [comment]
@rbgarga I signed the CLA 4 weeks ago and remerged the current master to solve the conflicst. Might you consider to merge this or are there any objections?

**Comment by oliwel** [comment]
@phil-davis @rbgarga 
Guys, is there any reason why this isnt processed? Can I assist in anything? I would really appreciated to see this in the official release

File: usr/local/bin/easyrule [hunk 2]
```diff
@@ -91,11 +91,24 @@ function is_specialnet($net) {
 
 
 if (($argc > 1) && !empty($argv[1])) {
+
+	# Automagically derive an alternate alias name from the scripts name
+	# This allows for using alternate alias lists with just a symlink
```
Comment (rbgarga) [inline usr/local/bin/easyrule]: Sorry about the long delay. Please just fix this comment style using /* */, that is recommended way for multiline comments and I'll get it merged

**Comment by oliwel** [comment]
done

**Pull request merged**